src/styles/_header.scss

```diff
@@ -25,6 +25,12 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+
+    transition: 0.3s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.